packages/driver/src/cy/actionability.ts

```diff
@@ -461,46 +460,24 @@ const verify = function (cy, $el, config, options, callbacks: VerifyCallbacks) {
   // make scrolling occur instantly. we do this by adding a style tag
   // and then removing it after we finish scrolling
   // https://github.com/cypress-io/cypress/issues/3200
-  const addScrollBehaviorFix = (element: JQuery<HTMLElement>) => {
-    const affectedParents: Map<HTMLElement, string> = new Map()
+  const addScrollBehaviorFix = () => {
+    let style
 
     try {
-      let parent: JQuery<HTMLElement> | null = element
+      const doc = $el.get(0).ownerDocument
 
-      do {
-        if ($dom.isScrollable(parent)) {
-          const parentElement = parent[0]
-          const style = getNativeProp(parentElement, 'style')
-          const styles = getComputedStyle(parentElement)
-
-          if (styles.scrollBehavior === 'smooth') {
-            affectedParents.set(parentElement, callNativeMethod(style, 'getStyleProperty', 'scroll-behavior'))
-            callNativeMethod(style, 'setStyleProperty', 'scroll-behavior', 'auto')
-          }
-        }
-
-        parent = $dom.getFirstScrollableParent(parent)
-      } while (parent)
+      style = doc.createElement('style')
+      style.innerHTML = '* { scroll-behavior: inherit !important; }'
+      // there's guaranteed to be a <script> tag, so that's the safest thing
+      // to query for and add the style tag after
+      doc.querySelector('script').after(style)
```

Comment:
Could we use adopted stylesheets to insert this rule without modifying the DOM itself?

Reply:
This was existing code that was reverted. I think this is something we could explore, but I'd rather revert the code to where it was previously and maybe explore this separately.